webpack/ForemanTasks/Components/TasksTable/Components/ConfirmationModals.js

```diff
@@ -1,49 +1,44 @@
 import React from 'react';
 import PropTypes from 'prop-types';
 import {
-  CANCEL_CONFIRM_MODAL_ID,
-  RESUME_CONFIRM_MODAL_ID,
-  CANCEL_SELECTED_CONFIRM_MODAL_ID,
-  RESUME_SELECTED_CONFIRM_MODAL_ID,
+  CONFIRM_MODAL,
+  CANCEL_MODAL,
+  RESUME_MODAL,
+  CANCEL_SELECTED_MODAL,
+  RESUME_SELECTED_MODAL,
 } from '../TasksTableConstants';
-import { CancelConfirm } from './CancelConfirm';
-import { ResumeConfirm } from './ResumeConfirm';
+import { ConfirmModal } from './ConfirmModal';
+
+const modalProps = type => {
+  if ([CANCEL_MODAL, CANCEL_SELECTED_MODAL].includes(type)) {
+    return { actionText: 'cancel', actionState: 'stopped' };
+  } else if ([RESUME_MODAL, RESUME_SELECTED_MODAL].includes(type))
+    return { actionText: 'resume', actionState: 'running' };
+  return { actionText: '', actionState: '' };
+};
```

Comment:
continue the discussion from https://github.com/theforeman/foreman-tasks/pull/538#discussion_r423713827
can we pass the right props from the parent component without adding extra login here?

Reply:
Added in a new commit `Refs #29597 - move modalProps to tasksTablePage`